app/plugins/statsd.md

```diff
@@ -54,18 +58,44 @@ parameter                     | default | description
 
 ## Metrics
 
-Plugin currently logs four metrics to the StatsD server: `request_count`, `request_size`, `response_size`, `status_count` and `latency`.
+Metrics the plugin supports logging to the StatsD server.
 
 Metric                     | description | namespace
----                        | ---         | -----
-`request_count`              | Increment the count of request made to the API by 1 | kong.\<api_name>.request.count
-`request_size`               | logs the request's body size in bytes | kong.\<api_name>.request.size
-`response_size`               | logs the response's body size in bytes | kong.\<api_name>.response.size
-`latency`                   | logs the time interval between the request started and response received from the upstream server | kong.\<api_name>.latency
-`status_count`               | For each status code returned, increment its counter by 1 | kong.\<api_name>.\<http_status_code>.count
-`unique_users`               | count of users made a request to the api | kong.\<api_name>.user.uniques
-`request_per_user`               | For each request by the user, increment its counter by 1 | kong.\<api_name>.\<consumer_id>.count
+---                        | ---         | ---
+`request_count`            | tracks api request | kong.\<api_name>.request.count
+`request_size`             | tracks api request's body size in bytes | kong.\<api_name>.request.size
+`response_size`            | tracks api response's body size in bytes | kong.\<api_name>.response.size
+`latency`                  | tracks the time interval between the request started and response received from the upstream server | kong.\<api_name>.latency
+`status_count`             | tracks each status code returned as response | kong.\<api_name>.status.\<status>.count and kong.\<api_name>.status.\<status>.total
```

Comment:
1) tracks each status code returned in `a` response
OR
2) tracks each status code returned `in every` response

It depends on what you mean.